Example-QueryInfo-REXX/README.md

```diff
@@ -1,13 +1,13 @@
 ## Example-QueryInfo-REXX
 
 This samples uses HWIREST API to:
-- List CPCs, filtered by the CPC name provided, in order to retrieve the URI and target name associated with that CPC
+- List CPCs and retrieves the URI and target name associated with that LOCAL CPC or the CPCname provided
 - Retrieve the following CPC information:
     - total memory installed
     - total memory available for LPARs
     - primary SE MAC
     - LPAR Resource Assignment
-- List the LPARs on that CPC, filtered by the LPAR name provided, in order to retrieve the URI and target name associated with that LPAR
+- List the LPARs on that CPC and retrieves the URI and target name associated with that LOCAL LPAR or the LPARname provided
```

Comment:
singlular vs plural

Reply:
i did a lot of copy and pastes, will correct

Reply:
fixed in the latest commit